### PR TITLE
Fix Plugin update + port to newer mkdocs

### PR DESCRIPTION
**Pull request by hjanuschka**
* rename from AvailablePlugins.md to available-plugins.md
  * ported to newer `mkdocs` - as mine didn't work straight after `pip install mkdocs`
  * one time generation of plugins.md
  * bundle update

for auto-update see follow-up PR: https://github.com/fastlane/fastlane/pull/7102

### CODE REVIEW

**Comment by hjanuschka** [comment]
@KrauseFx  i am also not sure, who calls the lane `update_available_plugins` do you do it manually?

File: fastlane/Fastfile [hunk 1]
```diff
@@ -8,6 +8,7 @@ lane :test do |options|
     ensure_assets_location
     ensure_tool_name_formatting
     ensure_code_samples
+    update_available_plugins
```
Comment (hjanuschka) [inline fastlane/Fastfile]: this is the actual change

File: fastlane/Fastfile [hunk 2]
```diff
@@ -18,7 +19,7 @@ lane :post_slack_deploy_message do
 end
 
 lane :update_available_plugins do
-  Fastlane::PluginFetcher.update_md_file!(output_path: "../docs/plugins/AvailablePlugins.md")
+  Fastlane::PluginFetcher.update_md_file!(output_path: "docs/plugins/available-plugins.md")
```
Comment (hjanuschka) [inline fastlane/Fastfile]: and that 👍

**Comment by fastlane-bot-helper** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>

    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><code>mkdocs.yml</code> was modified - make sure no modification was made by mistake</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

**Comment by hjanuschka** [comment]
@fastlane-bot-helper 
image

File: docs/plugins/available-plugins.md [hunk 3]
```diff
@@ -12,8 +12,96 @@ To search for a specific plugin
 fastlane search_plugins [search_query]
 ```
 
-You can find more information about how to start using plugins in [_fastlane_ Plugins](https://docs.fastlane.tools/plugins/create-plugin/).
```
Comment (KrauseFx) [inline docs/plugins/available-plugins.md]: Why this change?

File: docs/plugins/available-plugins.md [hunk 4]
```diff
@@ -12,8 +12,96 @@ To search for a specific plugin
...
+You can find more information about how to start using plugins in [Plugins.md](https://github.com/fastlane/fastlane/blob/master/fastlane/docs/Plugins.md).
 
 #### List of plugins
 
-The list of available plugins was moved to the [Actions#Plugins](https://docs.fastlane.tools/actions/#plugins)
```
Comment (KrauseFx) [inline docs/plugins/available-plugins.md]: No, let's leave it in actions.md

File: fastlane/Fastfile (hunk 2 above)
Comment (KrauseFx) [inline fastlane/Fastfile]: We don't use that file any more, do we?

**Comment by hjanuschka** [comment]
@KrauseFx feedback addressed, should the plugin stuff totaly be removed, tbh i don't fully understand the doc. building process right now. 😨

**Comment by KrauseFx** [comment]
Plugins are updated automatically on every fastlane release. It seems like this change is not necssary. Do you want to submit a PR just for the othe rchanges?

**Comment by hjanuschka** [comment]
@KrauseFx  will do a seperate PR on the other things
